src/vmm/tests/integration_tests.rs

```diff
@@ -50,10 +49,50 @@ fn wait_vmm_child_process(vmm_pid: i32) {
 }
 
 #[test]
-fn test_dummy_vmm() {
+fn test_dummy_vmm_elf() {
+    let pid = unsafe { fork() };
+    let vmm_config = VMMConfig {
+        kernel_config: default_kernel_config(None),
+        memory_config: default_memory_config(),
+        vcpu_config: default_vcpu_config(),
+    };
+    // Sanity check. Because the VMM runs in a separate process, if the file doesn't exist,
+    // all we see is a different exit code than 0.
+    assert!(vmm_config.kernel_config.path.as_path().exists());
+    match pid {
+        0 => {
+            set_panic_hook();
+            match VMM::try_from(vmm_config) {
+                Ok(mut vmm) => {
+                    vmm.run();
+                    // Shouldn't get here with this guest image. It will loop forever.
+                    unsafe { _exit(-1) };
```

Comment:
Is this still accurate? It looks like we are running with the halt image which means that it does not end here because of the way we are handling VcpuExit::Shutdown and VcpuExit::Hlt (i.e. with libc::exit).

I think it might make sense to actually just normally terminate the execution instead of using `libc::exit`. It would also make the testing simpler because we won't need to call fork. This last part is not in the scope of this PR, and we should handle it separately. But I think we should update the comment for now.

Reply:
Yes. If it ever gets to `exit(-1)`, something went wrong, because it's supposed to handle the `VcpuExit` by `exit(0)` earlier.

Reply:
I was commenting more on the "It will loop forever" part which seems inaccurate.